posts/2024-04-15_gno-tokyo/README.md

```diff
@@ -0,0 +1,65 @@
+---
+publication_date: 2024-04-15T00:00:00Z
+slug: gno-tokyo
+tags: [meetup, tokyo, community]
+authors: [leohhhn]
+---
+    
+# Intro to Gno Recap - Tokyo 2024
+
+[![Banner](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/banner.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/banner.png)
+
+A Gno.land meetup, "Intro to Gno", was held in Tokyo, Japan on the 11th of 
+April 2024. Attendees joined the meetup in one of Tokyo's largest centers, 
+Shinjuku City. The meetup was aimed at a local audience, with the slides being
+in Japanese. 
+
+The meetup focused on explaining Gno.land's major features:
+- Fully open-source smart-contracting
+- Custom smart contracting language for the future
+- Proof of Contribution
+
+[![points](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/three-points.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/three-points.png)
+#### Brief overview of gno.land in Japanese
+
+Gno.land's DevRel Engineer, Leon, covered these three topics separately, discussing the importance of such 
+features in open-source blockchain systems.
+
+[![leon-poc](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/leon-poc.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/leon-poc.png)
+#### @leohhhn with the PoC explanation slide in Japanese
+The Gno.land ecosystem was also introduced: Leon mentioned some of the major 
+projects in the Gno.land ecosystem - Gnoswap, Gnoscan, Adena, Gnosocial, and
+more.
+
+Following this, attendees were introduced to the Gno.land 
+[**Grants & Fund program**](https://github.com/gnolang/ecosystem-fund-grants),
+as well as a the **Game of Realms** program, presenting different ways builders 
+are incentivized to build in the ecosystem at this stage.
+
+Finally, attendees were given insight into the status of the project:
```

Comment:
**What is the future of Gno.land?**

Finally, we gave attendees a overview of the progress on Gno.land, from its beginnings to where we currently sit with regards to mainnet. We explained the progress made over the past year of building, and how Test4, an advanced experimental testnet prior to launch is important as it allows us to X, Y, and Z.  


We shared with attendees how they can get involved now, joining Discord, following the project on Twitter, contributing on GitHub and researching our Gno.land Grants Program.